third_party/move/move-compiler-v2/tests/acquires-checker/access_ok.exp

```diff
@@ -0,0 +1,121 @@
+
+Diagnostics:
+error: unnecessary acquires annotation
+  ┌─ tests/acquires-checker/access_ok.move:8:23
+  │
+8 │     fun f1() acquires S {
+  │                       ^^
+
+error: access specifier not enabled. Only plain `acquires R` is enabled.
+   ┌─ tests/acquires-checker/access_ok.move:11:20
+   │
+11 │     fun f2() reads S {
+   │                    ^^
+
+error: access specifier not enabled. Only plain `acquires R` is enabled.
+   ┌─ tests/acquires-checker/access_ok.move:14:21
+   │
+14 │     fun f3() writes S {
+   │                     ^^
+
+error: unnecessary acquires annotation
+   ┌─ tests/acquires-checker/access_ok.move:17:23
+   │
+17 │     fun f4() acquires S(*) {
+   │                       ^^^^
+
+error: unnecessary acquires annotation
```

Comment:
It seems like this annotation is not unnecessary if all other are not enabled.  Can we try to avoid inconsistent messages?

Reply:
We cannot distinguish `S(*)` from `S` in the model ast currently.
```rust
pub enum ResourceSpecifier {
    // ...
    Resource(QualifiedInstId<StructId>),
}
```
Both are represented by the struct id with empty instantiations.

Reply:
done